hack/gentestdb/gentestdb.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"flag"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"text/template"
+)
+
+func main() {
```

Comment:
extra space

Reply:
wrap this in 
```
func main(){
err:= run()
if err!=nil{
   panic(err)
}}
```
So we can have `return err` as a pattern like anywhere else in the code